client/src/index.js

```diff
@@ -22,31 +23,41 @@ const Routes = () => {
               Home
             </Menu.Item>
           </Link>
+
           <Link className="nav-link" to="/about">
             <Menu.Item name="editorials" active={true}>
               About
             </Menu.Item>
           </Link>
+
           <Link className="nav-link" to="/status">
             <Menu.Item name="editorials" active={true}>
               Status
             </Menu.Item>
           </Link>
+
           <Link className="nav-link" to="/sign-up-user">
             <Menu.Item name="editorials" active={true}>
               User Sign Up
             </Menu.Item>
           </Link>
+
           <Link className="nav-link" to="/login">
             <Menu.Item name="editorials" active={true}>
               Login
             </Menu.Item>
           </Link>
+
           <Link className="nav-link" to="/sign-up-interpreter">
             <Menu.Item name="editorials" active={true}>
               Become Interpreter
             </Menu.Item>
           </Link>
+          <Link className="nav-link" to="/text">
```

Comment:
This functionality should only be available to logged in user. Can you display this link only if the user is logged in? Also the route name can be "add-document" instead of "text" for clarity, and the text displayed be "Add document" instead of "Add text".

Reply:
done